.github/workflows/cron.yml

```diff
@@ -49,6 +49,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - if: matrix.os == 'macos-latest'
+        name: Set up hdf5
+        run: brew install hdf5
+
```

Comment:
that reminds me: I think I didn't put it in the cron of omc3

Reply:
True. Also, we should probably migrate the `omc3` CI to the common workflows as well.